src/screens/FeaturedSearchScreen.tsx

```diff
@@ -8,10 +12,36 @@ import SearchInput from '../components/SearchInput';
  * @returns {JSX.Element} | 'not logged in' search screen, the landing page of the app
  */
 export default function FeaturedSearchScreen(): JSX.Element {
+    const { profile, setProfile } = useProfileContext();
+    const [loading, setLoading] = useState<boolean>(true);
+    const [trendingShows, setTrendingShows] = useState<ShowData[] | null>(null);
+
+    useEffect(() => {
+        const handler = async () => {
+            const movieData: ShowData[] | null = await getMovieTrending();
+            const tvData: ShowData[] | null = await getTvTrending();
+            if (movieData && tvData) {
+                setTrendingShows([...movieData, ...tvData]);
+            }
+            setLoading(false);
+        };
+        handler();
+    }, []);
     return (
         <>
             <h1 data-testid='featured-search-heading'>Featured Search Page</h1>
             <SearchInput />
+
+            {loading ? (
+                <ShowCarouselPlaceholder count={5} />
+            ) : (
+                <ShowCarousel
+                    data={trendingShows}
+                    size={5}
```

Comment:
I see we are hardcoding the size of these carousels currently. Do we have a plan to dynamically update these on screen size change? I think we should create a ticket for this with 'TODO' comments as I imagine this currently is non-functional on mobile. Let me know if I am mistaken.

Reply:
Created https://github.com/Thenlie/Streamability/issues/458